src/FluentCMS.Web.UI.Components/Layouts/PanelLayout.razor

```diff
@@ -11,7 +11,7 @@
                 
                 <!-- TODO: Sidebar Toggler -->
 
-                <a href="/" class="flex items-center" id="adminNavbarLogoLink">
+                <a href="/adnmin" class="flex items-center" id="adminNavbarLogoLink">
```

Comment:
there is a typo in this pull request, i don't check the other change please check your pull requests before apply so we won't waste time thanks.

Reply:
somehow i missed it, thanks!